pkg/workflows/testutils.go

```diff
@@ -0,0 +1,46 @@
+package workflows
+
+import (
+	"bytes"
+	"testing"
+	"text/template"
+
+	"github.com/stretchr/testify/require"
+)
+
+// WFYamlSpec generates a validate yaml spec for a workflow for the given name and owner
+func WFYamlSpec(t *testing.T, name, owner string) string {
```

Comment:
this is a standalone utility because it's useful in core tests that need to create job specs

Reply:
Could you rename it to utils_test.go? That way it will only be included in tests?

Reply:
I'm not a huge fan of this refactoring, this feels like a case of "a little duplication is better than a little coupling" to me and we could add this to whatever package needs it.

Reply:
hmm, i started by copying then moved it here

 the packages are strongly coupled though. you can't do *anything useful* with a TOML spec without this package because the workflow job TOML spec depends on this pkg. given that, it seemed like this package ought to be responsible for providing some helpers.

said differently, if we copy this, and we change the yaml and/or validation in this repo, then we will not only need to bump the core repo, but also fix the embedded string literal to match the change.

wdyt @cedric-cordenier

Reply:
Discussed offline, I'm happy to go with this in the end